src/modules/Core/Account/settings.js

```diff
@@ -16,16 +17,14 @@ export const PASSWORD_RECOVERY_REMINDERS_SHOWN = {
   '20000': false,
   '200000': false
 }
-
-// TODO:  Remove hardcoded currency defaults
-// Default Account Settings
-export const SYNCED_ACCOUNT_DEFAULTS = {
-  autoLogoutTimeInSeconds: 3600,
-  defaultFiat: 'USD',
-  defaultIsoFiat: 'iso:USD',
-  merchantMode: false,
-  preferredSwapPluginId: '',
-  countryCode: '',
+type CurrencyDefault = {
+  [currency: string]: {
+    denomination?: string,
+    denomination?: string,
```

Comment:
I'm pretty sure one of them should be 'denominations', plural. 
How does it even let you commit this?